apps/web/pages/api/auth/signup.ts

```diff
@@ -12,6 +12,11 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
     return;
   }
 
+  if (process.env.NEXT_PUBLIC_DISABLE_SIGNUP) {
```

Comment:
We should do `if (process.env.NEXT_PUBLIC_DISABLE_SIGNUP === "true")` because as of the above code, any non empty string, including `"false"` is considered _truthy_.

Explanation: `!!"false"` and `Boolean("false")` is `true`.

Checking against the `true` string ensures backwards compatibility and that only users really intending to use this feature will disable signups.